client/src/pages/european-projects/components/pages/analysis/positioning/charts/top-10-beneficiaries/index.tsx

```diff
@@ -21,7 +21,7 @@ export default function Top10Beneficiaries() {
   return (
     <ChartWrapper
       id="top10beneficiaries"
-      options={options(data)}
+      options={options(data, searchParams.get('country_code') || null)}
```

Comment:
I now prefere to use Nullish coalescing operator https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing. What do you think about ?